airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -114,6 +116,13 @@ def _inner_wrapper(self: "GCSHook", *args, **kwargs) -> RT:
     return _wrapper
 
 
+# A fake bucket to use in functions decorated by provide_session. This allows
```

Comment:
`_fallback_object_url_to_object_name_and_bucket_name` I think you mean, not provide_session :)